app/views/appliances/show.html.haml

```diff
@@ -44,12 +48,12 @@
         - color = "text-green-500" if comment.category == "fix"
         - color = "text-red-500" if comment.category == "problem"
         .col-span-1.text-sm.uppercase.font-bold.py-1{class: "#{color}"}
-          = comment.category
+          = t comment.category
         .col-span-2.md:grid.md:grid-cols-2.py-1
           .text-sm.text-right.md:text-center
             = l comment.created_at, format: :short
           .text-sm.font-medium.text-blue-600.text-right
-            = link_to "Gyula bácsi", 'insert user path here', class: "hover:underline"
+            = link_to comment.owner.name, user_path(comment.owner), class: "hover:underline"
```

Comment:
Right now the user policy for showing the user page is set to only show it if its your own user id or you are an admin, and so in most cases the link is not working.

Reply:
I made the requested changes.
Originally I let the links fail the authorization because, the show page not yet designed.